io/net/bonding.py

```diff
@@ -92,6 +92,7 @@ def setUp(self):
             for ipaddr, interface in zip(self.ipaddr, self.host_interfaces):
                 networkinterface = NetworkInterface(interface, self.localhost)
                 try:
+                    networkinterface.remove_ipaddr(ipaddr, self.netmask)
```

Comment:
@abdhaleegit I think this will error out if the interface is not configured already. did this worked for fresh non-configured interfaces?

Reply:
@Naresh-ibm yes, I have added flush_ipaddr() function to avocado and will update the PR .. this helps me in 2 things
1. it will remove if there are multiple ip addr configured
2. command will not fail if no ip is configured

example
3: enP49156p1s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    link/ether 8a:33:56:bd:bb:00 brd ff:ff:ff:ff:ff:ff
[root@ltc-zz3-lp3 ~]# ip addr flush  dev enP49156p1s0
[root@ltc-zz3-lp3 ~]# echo $?
0

3: enP49156p1s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    link/ether 8a:33:56:bd:bb:00 brd ff:ff:ff:ff:ff:ff
    inet 115.10.10.197/24 scope global enP49156p1s0
       valid_lft forever preferred_lft forever

[root@ltc-zz3-lp3 network-scripts]# ip addr flush  dev enP49156p1s0
[root@ltc-zz3-lp3 network-scripts]# echo $?
0
3: enP49156p1s0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP group default qlen 1000
    link/ether 8a:33:56:bd:bb:00 brd ff:ff:ff:ff:ff:ff